### ISSUE TITLE
[Modification request] CTA banner to support a muted/secondary variant

### ISSUE THREAD

**Issue by nsolerieu** [primer/brand#752]
Currently the CTA banner component supports only a primary background color. We often end up customizing the background via one-off CSS and noticed that we are often using the muted bg color similarly to card.

See example here: https://github.com/orgs/github/projects/19671/views/1?pane=issue&itemId=79615610

Only the border allow to visually delineate the component which is limiting and sometime looks very heavy in dense pages

![Image](https://github.com/user-attachments/assets/ac43a708-5442-4332-887e-525e3aba042d)

**Comment by joshfarrant** [comment]
Thanks for opening this @nsolerieu 👋

I'd be keen to get @danielguillan's take on this

**Comment by danielguillan** [comment]
Yes, the CTA Banner component currently supports setting a background color using the `--brand-CTABanner-bgColor` custom property. We could consider adding a new `backgroundColor` property to align it with [BreakoutBanner](https://primer.style/brand/components/BreakoutBanner/react#component-props), which supports `default`, `subtle`, and custom background colors.

Should we make `subtle` the default background color as well? This change would make sense if we use it most often, but it could introduce a visual breaking change.

**Comment by nsolerieu** [comment]
I'm supportive of aligning with the BreakoutBanner to have default/subtle via `backgroundColor` prop

**Comment by joshfarrant** [comment]
It looks like this is sort of related to https://github.com/primer/brand/issues/756 too.

It might be worth looking at `CTABanner` backgrounds holistically and coming up with a solution that works for both background colour and background image. It seems like `BreakoutBanner` supports both, so we could just do the same as we've done there.

**Comment by nsolerieu** [comment]
@joshfarrant I'm also in favor to bring parity between `CTABanner` and `BreakoutBanner` background options
